spec/rails_helper.rb

```diff
@@ -90,6 +90,7 @@
     ensure
       DatabaseCleaner.clean_with(:truncation)
       Attachment.all.each(&:delete)
```

Comment:
I'm wondering if we could save some time with a `.rm_rf` call on `Attachment.pwd` here?

Reply:
Seems like a good recommendation for an aside PR.